includes/management-options-page.php

```diff
@@ -22,6 +22,26 @@ function oadueslookup_plugin_menu()
     add_options_page('OA Dues Lookup', 'OA Dues Lookup', 'manage_options', 'oadueslookup', 'oadueslookup_options');
 }
 
+add_action( 'admin_notices', 'oadueslookup_admin_notices' );
+function oadueslookup_admin_notices() {
+    $lookup_slug = get_option('oadueslookup_oldslug', 'it was not set');
+    if (!($lookup_slug == 'it was not set')) {
+        ?><div class="updated">
+        <div>
+        <p>Your OA Dues Lookup page at <a href="<?php echo esc_html(get_option("home")) . "/" . esc_html($lookup_slug) ?>"><?php echo esc_html(get_option("home")) . "/" . esc_html($lookup_slug) ?></a> was converted from a specially-handled URL to a real WordPress Page, which contains the <code>[oadueslookup]</code> shortcode for the form. You can now use that shortcode on any page to show the dues lookup form.</p>
+        </div>
+        <div style="float: right;"><a href="?dismiss=oadl_shortcode_update">Dismiss</a></div>
+        <div style="clear: both;"></div>
+        </div><?php
+    }
+}
+add_action( 'admin_init', 'oadueslookup_dismiss_admin_notices' );
+function oadueslookup_dismiss_admin_notices() {
+    if ( isset( $_GET['dismiss'] ) && 'oadl_shortcode_update' == $_GET['dismiss'] ) {
```

Comment:
@justdave I think you may want to use `array_key_exists` instead of `isset` and `===` instead of `==`.

Reply:
I concur, done.